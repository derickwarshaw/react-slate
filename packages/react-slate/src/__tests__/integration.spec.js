/* @flow */

import React from 'react';
import { View } from '../components';
import renderToString from '../renders/renderToString';

describe('in integration tests renderToString', () => {
  it('should apply backgroundColor to paddings', () => {
    expect(
      renderToString(
        <View style={{ padding: '2', backgroundColor: 'ansi-red' }}>
          Hello World
        </View>,
        {
          height: 6,
        }
      )
    ).toMatchSnapshot();
  });
});
