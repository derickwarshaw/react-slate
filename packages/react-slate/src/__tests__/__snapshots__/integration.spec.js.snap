// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`in integration tests renderToString should apply backgroundColor to paddings 1`] = `
"@[41m               @[49m                         
@[41m               @[49m                         
@[41m  Hello World  @[49m                         
@[41m               @[49m                         
@[41m               @[49m                         
                                        "
`;
